frontend/package.json

```diff
@@ -27,7 +29,7 @@
     "gen:theme-typings": "chakra-cli tokens src/theme/index.ts",
     "postinstall": "npm run gen:theme-typings",
     "start": "cross-env SKIP_PREFLIGHT_CHECK=true react-app-rewired start",
-    "build": "cross-env SKIP_PREFLIGHT_CHECK=true react-app-rewired build",
+    "build": "cross-env CI=false BUILD_PATH='../dist/frontend' INLINE_RUNTIME_CHUNK=false SKIP_PREFLIGHT_CHECK=true react-app-rewired build",
```

Comment:
lol can we add a comment here on why these options are required pls? we will definitely forget

Reply:
I wish we can add comments to package.json... I guess we can go with a hacky way as demonstrated in this [SO post](https://stackoverflow.com/a/45815391) and add a `scriptComments` key?

Reply:
Done in e2aacb7